pytest/lib/mocknet.py

```diff
@@ -101,45 +139,43 @@ def setup_python_environments(nodes, wasm_contract):
     pmap(lambda n: setup_python_environment(n, wasm_contract), nodes)
 
 
-def start_load_test_helper_script(script, index, pk, sk):
+def start_load_test_helper_script(script, node_account_id, pk, sk):
     return f'''
         cd {PYTHON_DIR}
-        nohup ./venv/bin/python {script} {index} "{pk}" "{sk}" > load_test.out 2> load_test.err < /dev/null &
+        nohup ./venv/bin/python {script} {node_account_id} "{pk}" "{sk}" > load_test.out 2> load_test.err < /dev/null &
```

Comment:
I’d sleep less worried if you’d use `shlex` to quote the arguments:

    import shlex

    return '''
        cd {dir}
        nohup ./venv/bin/python {script} {node_account_id} {pk} {sk} > load_test.out 2> load_test.err < /dev/null &
    '''.format(dir=shlex.quote(PYTHON_DIR),
               script=shlex.quote(script),
               node_account_id=shlex.quote(str(node_account_id)),
               pk=shlex.quote(pk),
               sk=shlex.quote(sk))

On the other hand that might be beyond the scope of this commit.